templates/Uwp/_comp/CaliburnMicro/Project.SplitView.TabbedNav.Page.Settings/ViewModels/ShellViewModel_postaction.cs

```diff
@@ -7,14 +7,15 @@ public class ShellViewModel : Screen
     {
         private void OnItemInvoked(WinUI.NavigationViewItemInvokedEventArgs args)
         {
-            //{[{
             if (args.IsSettingsInvoked)
             {
+//{--{
+                // Navigate to the settings page - implement as appropriate if needed
```

Comment:
I don't understand this comment.
It's describing what the following line does. Why would someone need to implement it? How would they implement it? If they've selected to add a settings page, why would it not be needed?

Reply:
This is the postaction for adding the setting page, the block you mention will remove the comment, the next block adds the implementation. 

Wasn't that what you suggested in https://github.com/microsoft/WindowsTemplateStudio/issues/3784#issuecomment-670929031 ?
> By default, it will include the comment // Navigate to the settings page - implement as appropriate if needed
> When adding the Settings Page this comment will be replaced with the appropriate call to Navigate.

Reply:
Oops, I misread it as adding the line, not removing it. All good then